### PR TITLE
INT B-19882 backend ghc update customers to be non-cac users

### PR DESCRIPTION
**Pull request by danieljordan-caci**
## Summary

With the future work of [E-05384](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=E-05384), we want to set the backend up to be able to consider the `cac_validated` value when an office user can update a customer so they can designate if they are a non-CAC holder or not. This will bypass the CAC registration when a customer initially signs into MilMove.

This PR does the following:
- updates swagger & backend to receive `cac_validated` value when performing the `GET` request to retrieve a customer's profile information
- updates swagger & backend to send `cac_validated` value when performing the `POST` request to update a customer's profile information
- updated relevant tests

### How to test

1. Fire up `client` and `server` and log in as a TOO
2. Head on over to `http://officelocal:3000/swagger-ui/ghc.html#/customer/getCustomer` or you can use Postman and query a service member of your choice. 
If they have `cac_validated` as `true` in the database, it should appear in the response body. If it is false, it will not appear (this is because we don't have `x-nullable` set up in swagger).
3. Now head on over to `http://officelocal:3000/swagger-ui/ghc.html#/customer/updateCustomer` and put in the `If-Match` as well as the service member's `id`, and have this as your body:
```
{
  "cac_validated": true
}
```
4. You should now get a successful response, meaning we can use this in the UI in future stories and send that through the response and the office users will be able to view and update a service member if they are a non-CAC user

### CODE REVIEW

**Review by CoryKleinjanCACI — APPROVED** [review]
Expected values are returned in all listed test scenarios and changes to the code look good.

File: pkg/handlers/ghcapi/internal/payloads/model_to_payload_test.go [hunk 1]
```diff
@@ -231,6 +231,55 @@ func (suite *PayloadsSuite) TestProofOfServiceDoc() {
...
+		Affiliation:          &affiliation,
+		PersonalEmail:        &email,
+		Telephone:            &phone,
+		ResidentialAddress:   &residentialAddress,
+		BackupMailingAddress: &backupAddress,
+		CacValidated:         cacValidated,
+	}
+
+	suite.Run("Success - Returns a ghcmessages Upload payload from Upload Struct", func() {
+		returnedShipmentAddressUpdate := Customer(&customer)
```
Comment (traskowskycaci) [inline pkg/handlers/ghcapi/internal/payloads/model_to_payload_test.go:277]: Could you name this something a little more descriptive for the situation? returnedShipmentAddressUpdate doesn't seem like it fits, to me. I could be missing something though!

Comment (danieljordan-caci) [reply pkg/handlers/ghcapi/internal/payloads/model_to_payload_test.go:277]: Whoops. Copy paste oopsie. Corrected.

**Review by traskowskycaci — APPROVED** [review]
Looks gooood!

**Pull request merged**